### PR TITLE
Fix user queries

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix https://github.com/astronomer/houston-api/issues/77

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
pls merge only together with https://github.com/astronomer/houston-api/pull/95

File: houston/queries.go [hunk 1]
```diff
@@ -306,7 +306,7 @@ var (
 		  id
 		  label
 		  description
-		  users {
+		  systemUsers {
```
Comment (schnie) [inline houston/queries.go]: @andriisoldatenko I think we'll want the `workspaceUsers` query here. `systemUsers` is intended to be called via system admins and requires those permissions. `workspaceUsers` should be the equivalent of the old one.

Comment (andriisoldatenko) [reply houston/queries.go]: @schnie fixed, do you know where i can test it?
cc / @petedejoy

**Comment by ashb** [comment]
I don't think this change is needed. The `workspace` type still has a `users` property on it, and that doesn't change even with Pete's PR95. It's just the top level `query { users(...) {}}` that changes, but I don't think we call that from the CLI.

**Comment by schnie** [comment]
Oh, you're totally right @ashb. I'll close this out.

**Pull request closed without merging**